src/lambda/models/samLambdaRuntime.ts

```diff
@@ -78,12 +78,14 @@ export function getRuntimeFamily(langId: string): RuntimeFamily {
  */
 export function getDefaultRuntime(runtime: RuntimeFamily): string | undefined {
     switch (runtime) {
+        // note: compareSamLambdaRuntime sorts from least value to greatest, to alphabetically sort runtimes
+        // thus, pull the last value since that will represent the greatest (and thus, newest) version
```

Comment:
Is this worth a 2-line comment? Seems obvious. Or the same idea can be expressed in 1 line:

    // Sorts is ascending order, so last() returns the newest runtime.

Alternatively if `compareSamLambdaRuntime()` had a docstring on it this comment would not be needed.